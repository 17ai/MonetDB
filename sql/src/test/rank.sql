select RANK() over () as foo from tables;
select RANK() over (PARTITION BY schema_id) as foo, schema_id from tables;
select RANK() over (PARTITION BY schema_id ORDER BY schema_id) as foo, schema_id from tables;
select RANK() over (ORDER BY schema_id) as foo, schema_id from tables;
