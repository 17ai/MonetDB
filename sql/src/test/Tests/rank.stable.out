stdout of test 'rank` in directory 'src/test` itself:


# 19:18:12 >  
# 19:18:12 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/scratch/rc/Linux-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB4:/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB4/lib:/ufs/niels/scratch/rc/Linux-x86_64/lib/MonetDB4/bin" --set "gdk_dbfarm=/ufs/niels/scratch/rc/Linux-x86_64/var/MonetDB4/dbfarm" --set "sql_logdir=/ufs/niels/scratch/rc/Linux-x86_64/var/MonetDB4/sql_logs" --set "xquery_logdir=/ufs/niels/scratch/rc/Linux-x86_64/var/MonetDB4/xquery_logs" --set mapi_port=33222 --set xrpc_port=48072 --set monet_prompt= --trace "--dbname=mTests_src_test" --dbinit=" module(sql_server);" ; echo ; echo Over..
# 19:18:12 >  

# Monet Database Server V4.16.3
# Copyright (c) 1993-2007, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 19:18:12 >  
# 19:18:12 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=33222 
# 19:18:12 >  

% sys. # table_name
% foo # name
% oid # type
% 2 # length
[ 0@0	]
[ 1@0	]
[ 2@0	]
[ 3@0	]
[ 4@0	]
[ 5@0	]
[ 6@0	]
[ 7@0	]
[ 8@0	]
[ 9@0	]
[ 10@0	]
[ 11@0	]
[ 12@0	]
[ 13@0	]
[ 14@0	]
[ 15@0	]
[ 16@0	]
[ 17@0	]
[ 18@0	]
[ 19@0	]
[ 20@0	]
[ 21@0	]
[ 22@0	]
[ 23@0	]
[ 24@0	]
[ 25@0	]
[ 26@0	]
[ 27@0	]
[ 28@0	]
[ 29@0	]
[ 30@0	]
[ 31@0	]
% sys.tables,	sys.tables # table_name
% foo,	schema_id # name
% oid,	int # type
% 2,	4 # length
[ 0@0,	4577	]
[ 1@0,	4577	]
[ 2@0,	4577	]
[ 3@0,	4577	]
[ 4@0,	4577	]
[ 5@0,	4577	]
[ 6@0,	4577	]
[ 7@0,	4577	]
[ 8@0,	4577	]
[ 9@0,	4577	]
[ 10@0,	4577	]
[ 11@0,	4577	]
[ 12@0,	4577	]
[ 0@0,	4676	]
[ 1@0,	4676	]
[ 2@0,	4676	]
[ 3@0,	4676	]
[ 4@0,	4676	]
[ 5@0,	4676	]
[ 13@0,	4577	]
[ 14@0,	4577	]
[ 15@0,	4577	]
[ 16@0,	4577	]
[ 17@0,	4577	]
[ 18@0,	4577	]
[ 19@0,	4577	]
[ 6@0,	4676	]
[ 20@0,	4577	]
[ 21@0,	4577	]
[ 22@0,	4577	]
[ 23@0,	4577	]
[ 24@0,	4577	]
% sys.tables,	sys.tables # table_name
% foo,	schema_id # name
% oid,	int # type
% 2,	4 # length
[ 0@0,	4577	]
[ 1@0,	4577	]
[ 2@0,	4577	]
[ 3@0,	4577	]
[ 4@0,	4577	]
[ 5@0,	4577	]
[ 6@0,	4577	]
[ 7@0,	4577	]
[ 8@0,	4577	]
[ 9@0,	4577	]
[ 10@0,	4577	]
[ 11@0,	4577	]
[ 12@0,	4577	]
[ 0@0,	4676	]
[ 1@0,	4676	]
[ 2@0,	4676	]
[ 3@0,	4676	]
[ 4@0,	4676	]
[ 5@0,	4676	]
[ 13@0,	4577	]
[ 14@0,	4577	]
[ 15@0,	4577	]
[ 16@0,	4577	]
[ 17@0,	4577	]
[ 18@0,	4577	]
[ 19@0,	4577	]
[ 6@0,	4676	]
[ 20@0,	4577	]
[ 21@0,	4577	]
[ 22@0,	4577	]
[ 23@0,	4577	]
[ 24@0,	4577	]
% .,	sys.tables # table_name
% foo,	schema_id # name
% oid,	int # type
% 2,	4 # length
[ 0@0,	4577	]
[ 1@0,	4577	]
[ 2@0,	4577	]
[ 3@0,	4577	]
[ 4@0,	4577	]
[ 5@0,	4577	]
[ 6@0,	4577	]
[ 7@0,	4577	]
[ 8@0,	4577	]
[ 9@0,	4577	]
[ 10@0,	4577	]
[ 11@0,	4577	]
[ 12@0,	4577	]
[ 24@0,	4676	]
[ 25@0,	4676	]
[ 27@0,	4676	]
[ 28@0,	4676	]
[ 29@0,	4676	]
[ 30@0,	4676	]
[ 31@0,	4577	]
[ 19@0,	4577	]
[ 20@0,	4577	]
[ 21@0,	4577	]
[ 22@0,	4577	]
[ 23@0,	4577	]
[ 16@0,	4577	]
[ 14@0,	4676	]
[ 15@0,	4577	]
[ 13@0,	4577	]
[ 17@0,	4577	]
[ 18@0,	4577	]
[ 26@0,	4577	]

# 19:18:12 >  
# 19:18:12 >  Done.
# 19:18:12 >  

